OsmAnd/src/net/osmand/plus/plugins/monitoring/MonitoringSettingsFragment.java

```diff
@@ -272,12 +272,18 @@ private void setupShowTripRecNotificationPref() {
 
 	public void setupObdRecordingPref() {
 		Preference preference = findPreference(RECORD_OBD_DATA);
+
+		@ColorRes int iconColor = isNightMode() ? R.color.icon_color_default_light : R.color.icon_color_default_dark;
```

Comment:
its ok, but annotations in local variables make code harder to read (if there are many of them)